requirements.dev.txt

```diff
@@ -1,5 +1,6 @@
 aiohttp==3.9.4
 black>=20.8b1
+bump2version==1.0.1
```

Comment:
you can take this back out; it doesnt get used anywhere in this repo except for those two gh actions files, where it gets installed w/o versioning by calls to `python -m pip install bump2version`

Reply:
cool, thanks for checking